### PR TITLE
Feature/feature calendar events

### PR DESCRIPTION
**Pull request by enrichp**

### CODE REVIEW

File: app/src/test/java/org/pesmypetcare/mypetcare/controllers/TestTrUpdatePetWash.java [hunk 1]
```diff
@@ -50,7 +55,7 @@ public void shouldUpdateWashBody() throws WashAlreadyExistingException {
 
     @Test
     public void shouldUpdateWashDate() throws WashAlreadyExistingException,
-            org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException {
+        org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException, ExecutionException, InterruptedException {
```
Comment (santidrj) [inline app/src/test/java/org/pesmypetcare/mypetcare/controllers/TestTrUpdatePetWash.java:58]: Codacy Issue found: [Unnecessary use of fully qualified name 'org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException' due to existing import 'org.pesmypetcare.usermanagerlib.exceptions.InvalidFormatException'](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5527577)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/meals/TrDeleteMeal.java  2
         

Complexity decreasing per file
==============================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/washes/TrDeleteWash.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5527577&bid=18116433)

File: app/src/main/java/org/pesmypetcare/mypetcare/services/MealManagerService.java [hunk 2]
```diff
@@ -31,28 +32,28 @@
      * @param newDate The new date of the meal
      * @param oldDate The old date of the meal
      */
-    void updateMealDate(User user, Pet pet, String newDate, String oldDate);
+    void updateMealKey(User user, Pet pet, String newDate, String oldDate) throws ExecutionException, InterruptedException;
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/services/MealManagerService.java:35]: Codacy Issue found: [Line is longer than 120 characters (found 123).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5527577)

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/controllers/meals/TrDeleteMeal.java  2
         

Complexity decreasing per file
==============================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/washes/TrDeleteWash.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5527577&bid=18116433)

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
Okey, I see no problem

**Pull request merged**